lua/neogit/lib/git/rebase.lua

```diff
@@ -107,6 +107,20 @@ function M.modify(commit)
   fire_rebase_event { commit = commit, status = "ok" }
 end
 
+function M.drop(commit)
+  local short_commit = string.sub(commit, 1, log.abbreviated_size())
```

Comment:
I forgot about this until now - you can use `git.rev_parse.abbreviate_commit(oid)` to get the correct short commit 😅

Reply:
Oh okay, then [this](https://github.com/NeogitOrg/neogit/commit/87b7fc6cfb516fe639687acf90f320f1f83321c9) is not very useful then. Should I remove it?

Reply:
yeah, that's fine :)